profile/README.md

```diff
@@ -0,0 +1,9 @@
+# SecureDNA
+
+SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
+- Current precompiled `synthclient` binaries are available for [Ubuntu amd64](https://github.com/SecureDNA/ppa).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
```

Comment:
Total nitpick: For something official/public, "source code" sounds more proper to my ears.
```suggestion
- Source code for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
```